test/Microsoft.IdentityModel.Tokens.Saml.Tests/SamlSecurityTokenHandlerTests.cs

```diff
@@ -116,6 +116,64 @@ public static TheoryData<SamlTheoryData> CanReadTokenTheoryData
         }
 
 #pragma warning disable CS3016 // Arrays as attribute arguments is not CLS-compliant
+        [Theory, MemberData("CreateClaimsIdentitysTheoryData")]
+#pragma warning restore CS3016 // Arrays as attribute arguments is not CLS-compliant
+        public void CreateClaimsIdentitys(SamlTheoryData theoryData)
+        {
+            TestUtilities.WriteHeader($"{this}.CreateClaimsIdentitys", theoryData);
+            var context = new CompareContext($"{this}.CreateClaimsIdentitys, {theoryData.TestId}");
+            try
+            {
+                var identities = ((theoryData.Handler) as DerivedSamlSecurityTokenHandler).CreateClaimsIdentitysPublic(theoryData.TokenClaimsIdentitiesTestSet.SecurityToken, theoryData.Issuer, theoryData.ValidationParameters);
+                theoryData.ExpectedException.ProcessNoException();
```

Comment:
pass context to ProcessNoException(context), this will give a better debug experience.
.